chapters/algebra-moonmath.tex

```diff
@@ -172,7 +172,7 @@ \subsection{The exponential map}
 \label{logarithm_map}
 log_g(\cdot): \G \to \Z_n\; x \mapsto log_g(x)
 \end{equation}
-Discrete logarithms are highly important in cryptography, because there are finite cyclic groups where the exponential map and its inverse, the discrete logarithm map, are believed to be one-way functions, which informally means that computing the exponential map is fast, while computing the logarithm map is slow (We will look into a more precise definition in \ref{crypto_groups}).
+Discrete logarithms are highly important in cryptography, because there are finite cyclic groups where the exponential map is believed to be a one-way function, which informally means that computing the exponential map is fast, while computing the logarithm map is slow (We will look into a more precise definition in \ref{crypto_groups}).
```

Comment:
Technically isn't the one that is "easy-to-compute" the one way function?
The inverse (the log) is not "one way" because is easy to invert (i.e. its inverse is the exponential)